lib/smart_proxy_host_reports/host_reports_api.rb

```diff
@@ -31,6 +31,10 @@ def check_content_type(format)
       log_halt(404, "Format argument not specified") unless format
       check_content_type(format)
       input = request.body.read
+      if Proxy::HostReports::Plugin.settings.incoming_save_dir
+        filename = File.join(Proxy::HostReports::Plugin.settings.incoming_save_dir, "#{format}-#{SecureRandom.uuid}")
```

Comment:
Not a request, but more like suggestion: maybe instead of UUID we could use a timestamp?

Reply:
Good point, sure.